tests/Microsoft.Bot.Builder.Azure.Tests/StorageEmulatorHelper.cs

```diff
@@ -45,7 +44,6 @@ public static bool CheckEmulator()
         {
             if (!string.IsNullOrEmpty(Environment.GetEnvironmentVariable("AGENT_NAME")))
             {
-                Assert.Inconclusive("This test requires Azure Storage Emulator to run and is disabled on the build server.");
```

Comment:
Is this behavior the same using xunit? Or should we find a way to throw this inconclusive result?

Reply:
We replaced this with the `IgnoreOnNoEmulatorFact` attribute. If the Storage Emulator is not installed and running, the tests are still skipped.